sample/src/main/assets/json.form/basic_form.json

```diff
@@ -353,6 +353,45 @@
           "user_two": "1713AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
         }
       },
+      {
+        "key": "response_spinner_with_options",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "spinner",
+        "hint": "Response Spinners",
+        "options": [
+          {
+            "key": "yes",
+            "text": "Yes",
+            "value": false,
+            "openmrs_entity": "",
+            "openmrs_entity_id": ""
+          },
+          {
+            "key": "no",
+            "text": "No",
+            "value": false,
+            "openmrs_entity": "",
+            "openmrs_entity_id": ""
+          },
+          {
+            "key": "maybe",
+            "text": "Maybe",
+            "value": false,
+            "openmrs_entity": "",
+            "openmrs_entity_id": ""
+          }
+        ],
+        "v_required": {
+          "value": "true",
+          "err": "Please enter response"
+        },
+        "openmrs_choice_ids": {
+          "user_one": "1107AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+          "user_two": "1713AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+        }
```

Comment:
we should also update this functionality to make sure the openmrs_entities can be read from the options if the options are available. Then we wouldn't need this block for spinners with options

Reply:
Are these currently extracted in the widget factory?

Reply:
And what do the `openmrs_choice_ids` correspond to when compared with checkbox `options`?

Reply:
They correspond to the key & the openmrs_entity_id

Reply:
Thanks.

Reply:
Done.